azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/b2c/AADB2CAutoConfiguration.java

```diff
@@ -29,22 +28,69 @@
 @EnableConfigurationProperties(AADB2CProperties.class)
 public class AADB2CAutoConfiguration {
 
-    private final AADB2CProperties b2cProperties;
+    private final AADB2CProperties properties;
```

Comment:
Why rename this here?

Reply:
As in b2c module, the prefix b2c may not provide additional information.